androidApp/src/main/java/com/example/tasky/android/login/viewmodel/LoginViewModel.kt

```diff
@@ -32,7 +36,18 @@ class LoginViewModel : ViewModel() {
         }
     }
 
+    private fun validateInput(): Boolean {
```

Comment:
This is a minor one, but the function's naming does not indicate to me that I should expect a result. A name like `isInputValid()` better communicates that it will return a result I must handle. 

Take your `login`function in this same class; it is an excellent example of a name that communicates its single responsibility of just performing login logic which I can call and forget about. It would be a bit odd to receive a boolean from this function, don’t you think so?